qa/rpc-tests/p2p-segwit.py

```diff
@@ -4,15 +4,93 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test segwit transactions and blocks on P2P network."""
 
-from test_framework.mininode import *
-from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import *
-from test_framework.script import *
-from test_framework.blocktools import create_block, create_coinbase, add_witness_commitment, WITNESS_COMMITMENT_HEADER
-from test_framework.key import CECKey, CPubKey
-import time
-import random
 from binascii import hexlify
+import random
+import struct
+import time
+
+from test_framework.blocktools import (create_block,
+                                       create_coinbase,
+                                       add_witness_commitment,
+                                       WITNESS_COMMITMENT_HEADER)
+from test_framework.key import (CECKey,
+                                CPubKey)
+from test_framework.mininode import (CBlock,
+                                     CBlockHeader,
+                                     CInv,
+                                     COutPoint,
+                                     CTransaction,
+                                     CTxIn,
+                                     CTxInWitness,
+                                     CTxOut,
+                                     CTxWitness,
+                                     MAX_BLOCK_BASE_SIZE,
+                                     MSG_WITNESS_FLAG,
+                                     NODE_NETWORK,
+                                     NODE_WITNESS,
+                                     NetworkThread,
+                                     NodeConn,
+                                     NodeConnCB,
+                                     bytes_to_hex_str,
+                                     hash256,
+                                     hex_str_to_bytes,
+                                     mininode_lock,
+                                     msg_block,
+                                     msg_getdata,
+                                     msg_headers,
+                                     msg_inv,
+                                     msg_ping,
+                                     msg_pong,
+                                     msg_tx,
+                                     msg_witness_block,
+                                     msg_witness_tx,
+                                     ser_uint256,
+                                     ser_vector,
+                                     sha256,
+                                     uint256_from_str)
```

Comment:
This is fine, but the import list is super long. Did you use a tool to generate this? Wondering if it would be difficult to just replace all this with `from test_framework import mininode` and then add `mininode.` qualifiers where needed.

Alternately, maybe different formatting would be preferable. I've been using [yapf](https://github.com/google/yapf) for new code I write, which gives

```
+from test_framework.script import (CScript, CScriptNum, CScriptOp, CTransaction, CTxOut, OP_0, OP_1, OP_16, OP_2DROP,
+                                   OP_CHECKMULTISIG, OP_CHECKSIG, OP_DROP, OP_DUP, OP_ELSE, OP_ENDIF, OP_EQUAL,
+                                   OP_EQUALVERIFY, OP_HASH160, OP_IF, OP_RETURN, OP_TRUE, SIGHASH_ALL,
+                                   SIGHASH_ANYONECANPAY, SIGHASH_NONE, SIGHASH_SINGLE, SegwitVersion1SignatureHash,
+                                   SignatureHash, hash160, hash256, ser_uint256, sha256, uint256_from_str)
```

with column_limit=119.